src/devices/Ssd13xx/samples/Program.cs

```diff
@@ -2,116 +2,248 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
-using System.Collections.Generic;
+using System.Threading.Tasks;
 using System.Device.I2c;
 using System.Drawing;
-using System.Linq;
-using System.Net;
-using System.Net.NetworkInformation;
-using System.Net.Sockets;
+using System.Device.Spi;
 using System.Threading;
 using System.IO;
+using System.Linq;
+using System.Device.Gpio;
 using Iot.Device.Arduino;
 using Iot.Device.Graphics;
 using Iot.Device.Graphics.SkiaSharpAdapter;
 using Iot.Device.Ssd13xx;
+using Iot.Device.Board;
+using Ssd13xx.Samples.Simulations;
 
-/// <summary>
-/// Demo entry point
-/// </summary>
-public class Program
+namespace Ssd13xx.Samples
 {
     /// <summary>
-    /// Entry point
+    /// Demo entry point
     /// </summary>
-    public static int Main(string[] args)
+    public class Program
```

Comment:
rather than adjusting this sample to try to cover all the cases, maybe it's better to have another sample focusing on this new screen?

Reply:
I grappled with this a bit but that's definitely the better approach 👍

Reply:
I restructured the sample projects to be broken out by protocol. The diff is a bit messy as a result but this is the folder structure:
![image](https://github.com/dotnet/iot/assets/56238104/e579a830-ae00-4d3f-843b-b3faa73d81e7)